src/transformer/cli/action/transformer.action.ts

```diff
@@ -103,11 +103,7 @@ const QPVFromDataGouv = async (): Promise<QpvTransfer[]> =>
   (await axios.get('https://www.data.gouv.fr/fr/datasets/r/14caff6e-2619-4127-8518-0c33560c5eb4')).data;
 
 const ZRRFromEquipementsSportsGouv = async (): Promise<ZrrTransfer[]> =>
-  (
-    await axios.get(
-      'https://equipements.sports.gouv.fr/api/explore/v2.1/catalog/datasets/insee-zrr/exports/json?select=zrr_simp&refine=zrr_simp:"C - Classée en ZRR"'
-    )
-  ).data;
+  (await axios.get('https://equipements.sports.gouv.fr/api/explore/v2.1/catalog/datasets/insee-zrr/exports/json')).data;
```

Comment:
```suggestion
  (
  (await axios.get('https://equipements.sports.gouv.fr/api/explore/v2.1/catalog/datasets/insee-zrr/exports/json')).data;
    await axios.get(
      'https://equipements.sports.gouv.fr/api/explore/v2.1/catalog/datasets/insee-zrr/exports/json?select=zrr_simp&refine=zrr_simp:"C - Classée en ZRR"'
    )
  ).data;
```
Pourquoi avoir enlevé le filtre sur zrr_simp ?

Reply:
Sans le filtre on télécharge 8.6Mb de données et avec on ne télécharge que 707.8kb de données et il s'agit seulement des données qui nous intéressent à quelques exceptions près